apps/meteor/app/apps/server/converters/messages.js

```diff
@@ -243,6 +244,14 @@ export class AppMessagesConverter {
 		);
 	}
 
+	async convertSystemMessage(message) {
+		if (!message) {
+			return undefined;
+		}
+
+		return this.convertMessageRaw(message, { type: 't' });
```

Comment:
Preferably use the `convertMessage` method. This one is kind of reserved for "bulk" reading of messages, because it doesn't go to the database to fill the relationships with the full data.